src/Blob.mo

```diff
@@ -1,53 +1,199 @@
-/// Binary blobs
-
+/// Module for working with Blobs: immutable sequence of bytes.
+///
+/// Blobs represent bytes. They are immutable, iterable, but not indexable.
+///
+/// Blobs are also often represented as `[Nat8]`, i.e. an array of bytes.
+/// If you would like to manipulate Blobs, it is recommended that you convert
+/// Blobs to `[var Nat8]` or `Buffer<Nat8>`, do the manipulation, then convert back.
+///
+/// Import from the base library to use this module.
+/// ```motoko name=import
+/// import Blob "mo:base/Blob";
+/// ```
+///
+/// Some built in features not listed in this module:
+/// ```motoko include=import
```

Comment:
```suggestion
/// Some built in features not listed in this module:
/// 
/// * You can create a `Blob` literal from a `Text` literal, provided the context expects an expression of type `Blob`.
/// * `b.size() : Nat` returns the number of bytes in the blob `b`;
/// * `b.vals() : Iter.Iter<Nat8>` returns an iterator to enumerate the bytes of the blob `b`.
///
///  For example:
/// ```motoko include=import
```

Reply:
(I'm not sure what the allowed char in the text literal are, but I think you can have (ASCII?) chars as well as escaped hex.)